crates/iota-network/src/randomness/mod.rs

```diff
@@ -982,6 +982,14 @@ impl RandomnessEventLoop {
                     continue; // don't send partial sigs to self
                 }
                 let mut client = RandomnessClient::new(peer.clone());
+                // `test_byzantine_peer_handling` built in debug mode takes
+                // longer to verify invalid signatures and thus needs larger
+                // timeouts.
+                #[cfg(test)]
+                const SEND_PARTIAL_SIGNATURES_TIMEOUT: Duration = Duration::from_secs(500);
```

Comment:
500 sec [seems to work on self-hosted-arm64](https://github.com/iotaledger/iota/actions/runs/13791724155/job/38573592500#step:9:2993).